core/src/main/kotlin/fr/sncf/osrd/standalone_sim/etcsContextBuilder.kt

```diff
@@ -67,11 +67,55 @@ fun buildETCSDangerPoints(
     return res.sorted()
 }
 
+/**
+ * Builds the list of ETCS detectors on the path. Goes until the last detector before the end of the
+ * path or, if the last block is ETCS, until the first ETCS detector after the end of the path.
```

Comment:
If I understood the code correctly and the purpose matches the code (only exit detectors):
```suggestion
 * Builds the offset list of exit detectors for ETCS-blocks on the block-path. Goes until the last block-transition before the end of the
 * travelled-path or, if the last block is ETCS, until the first ETCS-block exit-detector after the end of the travelled-path.
```

Reply:
Didn't exactly go for that, only used `-` when there could be a confusion, so at the end of the sentence.
See https://github.com/OpenRailAssociation/osrd/pull/12457/commits/7af9ad6e3d7d9fe63a4f903a612488d5ae47b3fe.